components/elasticsearch/scripts/start.py

```diff
@@ -70,4 +70,5 @@ def main():
         # created in the 'create.py' script.
         es_upgrade_utils.restore_upgrade_data(es_endpoint_ip, ES_ENDPOINT_PORT)
 
+
```

Comment:
Not sure what version of flake8 is running in the travis CI environment, but as you can see above, the static analysis failed for 4c529cc.